pkgs/applications/graphics/rapid-photo-downloader/default.nix

```diff
@@ -19,24 +19,31 @@ python3Packages.buildPythonApplication rec {
       --replace "disable_version_check = False" "disable_version_check = True"
     substituteInPlace raphodo/rescan.py \
       --replace "from preferences" "from raphodo.preferences"
-    substituteInPlace raphodo/copyfiles.py \
-      --replace "import problemnotification" "import raphodo.problemnotification"
   '';
 
-  nativeBuildInputs = [ file intltool gobject-introspection ];
+  nativeBuildInputs = [
+    exiftool
+    file
+    intltool
+  ];
 
   buildInputs = [
-    libgudev
-    udisks
-    glib
-    gnome3.gexiv2
-    gst_all_1.gstreamer
-    libnotify
-    exiv2
     exiftool
-    qt5.qtimageformats
     gdk_pixbuf
-  ] ++ (with python3Packages; [
+    gexiv2
+    gobject-introspection
+    gst_all_1.gst-libav
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-good
+    gst_all_1.gstreamer
+    gst_all_1.gstreamer.dev
+    libgudev
+    libmediainfo
```

Comment:
`libmediainfo` is specified in `makeWrapperArgs` and can thus be removed from here

Reply:
Done